docs/md/docs/2020-07-09-last-dance.md

```diff
@@ -0,0 +1,96 @@
+# The Last (Engineering) Dance
+
+Watching `The Last Dance` was an incredible experience, much more than watching Michael Jordan or specifically to sports - I treated this series as a team building tutorial.<br>
+So many team building and personal growth topics were raised there, I just found myself sitting after every episode and thinking on the episode's relevancy to my work as team lead and what I can learn from it.
+
+![The Chicago Bulls](/docs/assets/last-dance/group-header.jpeg)
+
+## tl;dr
+
+This blog post is about taking lessons from `The Last Dance` and applying them on the engineering world. <br>
+I'll focus on the processes that took place in the Chicago Bulls that converted them from a great team to the best group of their time.
+A lot can be learned from the different approaches that the coaches took and the different behaviors of Michael Jordan that led to this improvement in the team's performance.
+
+## Comparing a basketball team with an engineering team
+
+I'll begin with doing a basic mapping (as I see it) between the series characters to an engineering org:
+* The coach (Phil Jackson and Doug Collins before him) - the team lead
+* Michael Jordan - a senior engineer
+* The rest of the Chicago Bulls team - engineers in the team
+* Jerry Reinsdorf and the rest of the Chicago Bulls mgmt - different stakeholders, "upper management"
+* The fans and the crowd - customers
+
+Now that we have this comparison in mind we can begin with the real fun :)
+
+## The team lead's approach to winning
+The coach/team lead has so much on their heads:
+* Satisfying all stakeholders inside the organization - bringing money in by winning/selling
+* Satisfying the fans/customers in the long term
+* Satisfying the team itself - personal growth, ego, salaries
+* Satisfying themselves - let's not forget that the coach/team lead wants to develop and enjoy as well
+<br>
+Let's compare the two approaches of the two coaches in the series:
+
+### Doug Collins - win by maximizing reliance on Michael Jordan
+Doug Collins was a coach that brought great results to the team by putting all the focus on Michael Jordan, all the game moves and tactics was built on Michael Jordan's abilities.
+This is an example to a team lead that puts the weight of the team's ability to deliver only on the senior engineers in the team:
+* Tough and complicated tasks are assigned to the senior engineers
+* The non-senior engineers' purpose is to let the senior engineers run fast and deliver
+
+This made the Chicago Bulls a great team because they had one hell of a senior engineer :) but they could not win the championship.
+There are a lot of disadvantages to this approach:
+* The team's ability to deliver is equal to the senior engineers' ability to deliver
+* The ability to improve is equal to the ability of your specific senior engineers to improve
```

Comment:
should `your` be part of the sentence? the write-up does not mention any belonging to the reader itself